azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/CallCompositeActivity.kt

```diff
@@ -146,19 +143,14 @@ internal class CallCompositeActivity : AppCompatActivity() {
     }
 
     private fun configureLocalization() {
+        val config: Configuration = resources.configuration
         configuration.localizationConfig?.let { localeConfig ->
             localeConfig.layoutDirection.let {
                 window?.decorView?.layoutDirection = it
             }
-            val config: Configuration = resources.configuration
-            val languageAttributes = localeConfig.languageCode.toString().split("-")
-            val languageCode = languageAttributes[0]
-            val countryCode =
-                if (languageAttributes.size > 1) languageAttributes[1] else ""
-            config.setLocale(Locale(languageCode, countryCode))
-            resources.updateConfiguration(config, resources.displayMetrics)
-            localizationProvider.apply(localeConfig)
+            config.setLocale(localeConfig.locale)
         }
+        resources.updateConfiguration(config, resources.displayMetrics)
```

Comment:
This is deprecated in API 25, please take a look at how to handle it for API 25 and newer

Reply:
Yea, there is no easy and new native way to replace this method.  Will leave it for now. 

https://stackoverflow.com/questions/39705739/android-n-change-language-programmatically/40849142#40849142